src/Forms/SearchableDropdownTrait.php

```diff
@@ -450,9 +457,16 @@ public function getSchemaDataDefaults(): array
 
     public function getSchemaStateDefaults(): array
     {
-        $data = parent::getSchemaStateDefaults();
```

Comment:
We should still be calling parent::getSchemaStateDefault() and merging it the new data here. There's a stuff that's added in SingleSelectField::getSchemaDataDefaults() and FormField::getSchemaDataDefaults()

Reply:
The whole point of this PR is to avoid calling `getSource()` which we can't do if we call `parent::getSchemaDataDefaults()`.

There's nothing happening in `FormField::getSchemaDataDefaults()` or `SingleSelectField::getSchemaDataDefaults()` which matters and isn't happening here.
If you really want I'll copy exactly everything that's happening in those classes' methods into here (assuming I didn't already, I can't remember) - but that's not necessary, the field works as it should like this.

Reply:
Ah yup, sorry I got confused by `FormField::getSchemaDataDefaults()` (I assume your reply you meant getSchemaStateDefaults(), though you wrote getSchemaDataDefaults()`) - yeah there's nothing in the the ancestors for getSchemaDataDefaults() except for 'source' which we don't want here.